Ansible/ansible_collections/jfrog/platform/roles/artifactory/defaults/main.yml

```diff
@@ -69,12 +69,21 @@ artifactory_admin_password: password
 artifactory_service_file: /lib/systemd/system/artifactory.service
 
 # Provide binarystore XML content below with 2-space indentation
+# For HA
 artifactory_binarystore: |-
```

Comment:
@bbaassssiiee Thanks for PR, does it  not make sense to have single name as `artifactory_binarystore` and fix on chain template based on number of servers ? (avoid another variable `artifactory_filestore`)

Reply:
We can select the appropriate XML in the template based on `artifactory_ha_enabled`.

Reply:
We can select the appropriate XML in the template based on `artifactory_ha_enabled`.